README.md

```diff
@@ -1,57 +1,8 @@
-# Northcoders House of Games API
+In order to successfully conenct to the two databases locally you will need to add the following files;
 
-## Background
+.env-development
+.env-test
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+Within each file you will need to add the relevant database name ie PGDATABASE=<databasename>
```

Comment:
If you want someone to be able to use the repo (such as a potential employer , you will need to specify the names of each of the databases you have available)

Reply:
readme updated